styleguide/components/31-molecules/opening-hours/day-widget/_day-widget.scss

```diff
@@ -3,29 +3,54 @@ $breakpoint-openinghours: '(min-width: 750px)';
 // ===
 // Day view
 // ===
-.opening-hours-wrapper {
+.openinghours-widget {
   &[data-type="day"] {
     display: inline-block;
     border-radius: border-radius('radius-1');
     overflow: hidden;
 
-    .openinghours {
-      padding: .3rem .5rem 0 1.9rem;
-      background-color: color('green', -4);
+    > * {
       line-height: 1.4rem;
```

Comment:
I don't exactly know the context for this... but is this wise?

Reply:
The use case is as follows: 
- The markup inside the `.openinghours` comes from an external API, so we don't have any influence on that
- That markup contains block-level elements with the data for the opening hours, but that data is actually part of a sentence. A simplified example:
```html
<div class="openinghours">
    <div class="openinghours--date">
        <span>Today, </span>
        <span>20th of december</span>
    </div>
    <div class="openinghours--content">
          <span class="openinghours--status">open</span>
          <div class="openinghours--time">
              <span>from</span>
              <time>10:00</time>
              <span>to</span>
              <time>19:00</time>
          </div>
    </div>
</div>
``` 

- In the design, all this content is displayed inline
![screenshot 2018-12-20 at 14 45 44](https://user-images.githubusercontent.com/4415097/50288341-f62f1700-0465-11e9-8b9a-8f7257ee958f.png)

Visually speaking, the content is displayed in 1 sentence. So you tell me: is this wise? 😆
I'd say this is a valid use case to use this since the markup coming from the API is predefined.